grass7/vector/v.rast.bufferstats/v.rast.bufferstats.py

```diff
@@ -608,6 +634,18 @@ def main():
         # write results to file
         out.close()
 
+    if remove:
+        print("in remove")
```

Comment:
This can be removed or a GRASS message can be used?

Reply:
Removed and improved the next sentence.